test/gtest/unit_conv_solver.hpp

```diff
@@ -32,6 +32,12 @@
 #include "unit_conv_ConvolutionDescriptor.hpp"
 #include "unit_TensorDescriptor.hpp"
 
+#ifdef WORKAROUND_SWDEV_503936
```

Comment:
```suggestion
#if WORKAROUND_SWDEV_503936
```
#ifdef is always evaluated to TRUE, even if WORKAROUND_SWDEV_503936 is 0